src/main/resources/templates/receiverTemplate.hbs

```diff
@@ -34,14 +44,14 @@
                     schema="{{adapterRefs.schema}}"
                     root="{{adapterRefs.root}}"
                     {{#if adapterRefs.responseRoot}}responseRoot="{{adapterRefs.responseRoot}}"{{/if}}
-                    deepSearch="true">
+                    deepSearch="false">
                 {{#each parameters}}<Param name="{{this}}" sessionKey="{{this}}"/>{{/each}}
 
-                <Forward name="success" path="Build_query"/>
-                <Forward name="parserError" path="NotWellFormed"/>
-                <Forward name="illegalRoot" path="IllegalRoot"/>
-                <Forward name="failure" path="InvalidXml"/>
-            </Json2XmlInputValidator>
+                    {{#if forwards.success}}<Forward name="success" path="Build_query"/>{{/if}}
+                    {{#if forwards.parserError}}<Forward name="parserError" path="NotWellFormed"/>{{/if}}
+                    {{#if forwards.illegalRoot}}<Forward name="illegalRoot" path="IllegalRoot"/>{{/if}}
+                    {{#if forwards.failure}}<Forward name="failure" path="InvalidXml"/>{{/if}}
```

Comment:
Ik denk dat de forwards hier versimpeled kunnen worden (hetzelfde geldt voor de sender). 

De [documentatie](https://frankdoc.frankframework.org/#/All/Json2XmlValidator) zegt dat `failure` gebruikt moet worden wanneer de input niet aan de XSD voldoet. Hij roept nu een niet bestaande pipe of exit genaamd `InvalidXml` aan. Het zou leuk zijn als hij automatisch een 400 BadRequest teruggeeft.

`illegalRoot` bestaat niet en `parserError` wordt alleen aangeroepen wanneer de XSD niet correct is (ik ga er van uit dat deze tool altijd correcte XSDs produceert). De `success` forward mag van mij ook weg, omdat je dan automatisch naar de volgende pipe gaat (er bestaat nu geen pipe met naam `Build_query`)